data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+				to display
+					"reputation: Hai (Unfettered)" < -1250
+			label "tell me"
+			`	Suddenly, you hear a loud roar coming from the sea behind you, and the ground begins to shake.`
+			choice
+				`	(Turn around.)`
+			`	You see the waves become agitated and the wind rise as a shape surrounded by fumes emerges out of the water, as if an ancient sea monster was rising from the depths.`
+			`	But it is no monster. It's a ship, and its guns are pointed straight at you!`
+			label "try again"
+			scene "scene/emerging ladybug"
+			choice
+				`	(Run!)`
+				`	(Wait.)`
+					goto wait
+				`	(Try to contact my fleet for help.)`
+					to display
+						"total ships" > 1
+				`	(Shoot it!)`
+					goto foolish
+			`	As you try to react, the Hai closest to you firmly grabs your arm and shouts through the noise. "Wait! If this ship wanted to kill you, <last>, you would already be dead."`
+			label wait
+			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, one more at ease than the other. "This ship means you no harm."`
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"Why didn't you say anything before that happened?"`
+			`	The second one replies, "We could have, but where's the fun in that? We wanted to see how you would react.`
+			label explain
+			`	"This ship is our guarantee that nothing goes wrong, should you attempt something against us. After all, we are still not sure if we can trust you."`
+			`	The two Hai then take you to a nearby cave, showing you another ship. The first one speaks again. "That ship outside is not for sale. This ship, however, is a mighty relic of the past, intended for war from the times our empire used to control the stars - an Anomalocaris!"`
+			scene "scene/cave scene"
+			`	The second one now takes a step toward you to speak, almost interrupting the first. "We have a proposition for you: should you agree to bring much needed supplies to our brethren, the Unfettered Hai, we will sell you this ship for the modest price of 10 million credits, including 2 weapons of the savage Korath!" The first one seems surprised by this and tries to intervene, but the second resumes before they can speak up. "These legendary weapons can interfere with nearly any enemy engine system! Worry not, you'll be able to recuperate the cost of this ship with ease by bringing medical supplies to the Unfettered. This cargo is needed on short notice, on <destination>, and we lack the time and personnel to bring them.`
+			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
+			choice
+				`	"I'll take the... Anomalocaris, was it?"`
```

Comment:
```suggestion
				`	"I will help you."`
```
The player isn't choosing to take the ship, they're choosing to assist the Uhai, with the ship being a reward after the thought.